lib/crawler_web/controllers/user_auth_controller.ex

```diff
@@ -0,0 +1,94 @@
+defmodule CrawlerWeb.UserAuthController do
```

Comment:
As I remember, from the `mix phx.auth gen` this doesn't have the Controller, it's just `UserAuth` can you double check? 

https://github.com/topnimble/elixir-internal-certification/blob/develop/lib/elixir_internal_certification_web/controllers/user_auth.ex

Reply:
@andyduong1920 I fixed based on this [comment](https://github.com/vnntsu/elixir-ic/pull/32#discussion_r1042895502) from @Nihisil. I'm not sure if we should add the suffix `controller` or not. What do you guys think?

Reply:
This is not a controller @vnntsu cc @Nihisil it doesn't have any HTTP action and no Routes linked to that, you can double check it by running the `mix phx.auth gen` again 👌

Reply:
If it is not controller, it shouldn't be in the controller folder

Reply:
Sorry for confusion, I think it is ok to rename it back to `UserAuth` and keep it as is.

Reply:
Reverted in [d400c62](https://github.com/vnntsu/elixir-ic/pull/32/commits/d400c623567d60be79702d9fd20b2441a18ec00b)